harness/determined/cli/experiment.py

```diff
@@ -318,6 +318,28 @@ def describe(args: Namespace) -> None:
     render.tabulate_or_csv(headers, values, args.csv, outfile)
 
     # Display step-related information.
+    all_workloads: Dict[int, Dict[int, List[bindings.v1WorkloadContainer]]] = {}
+    for exp in exps:
+        if exp.id not in all_workloads:
+            all_workloads[exp.id] = {}
+        for trial in trials_for_experiment[exp.id]:
+            if trial.id not in all_workloads[exp.id]:
+                all_workloads[exp.id][trial.id] = []
+            # paginated read of all workloads in this trial
```

Comment:
there's `limit_offset_paginator` in this file which I believe you wrote, how about using it instead?

Reply:
:+1: got it